src/mail/view/MailViewer.js

```diff
@@ -295,37 +304,31 @@ export class MailViewer {
 							this._suspicious
 								? m(Banner, {
 									type: BannerType.Warning,
-									title: lang.get("phishingMessage_label"),
-									message: lang.get("phishingMessageBody_msg"),
+									title: "phishingMessage_label",
+									message: "phishingMessageBody_msg",
 									icon: Icons.Warning,
-									helpLink: lang.getInfoLink("phishing_link"),
-									buttons: [{text: lang.get("markAsNotPhishing_action"), click: () => this._markAsNotPhishing()}]
+									helpLink: "phishing_link",
+									buttons: [{text: "markAsNotPhishing_action", click: () => this._markAsNotPhishing()}]
 								})
 								: !this._warningDismissed && mail.authStatus === MailAuthenticationStatus.HARD_FAIL
 								? m(Banner, {
 									type: BannerType.Warning,
-									title: lang.get("mailAuthFailed_label"),
-									message: lang.get("mailAuthFailed_msg"),
+									title: "mailAuthFailed_label",
+									message: "mailAuthFailed_msg",
 									icon: Icons.Warning,
-									helpLink: lang.getInfoLink("mailAuth_link"),
-									buttons: [{text: lang.get("close_alt"), click: () => this._warningDismissed = true}]
-								})
-								: !this._warningDismissed && mail.authStatus === MailAuthenticationStatus.SOFT_FAIL
-									? m(Banner, {
-										type: BannerType.Info,
-										title: lang.get("mailAuthMissing_label"),
-										message: mail.differentEnvelopeSender ? lang.get("technicalSender_msg", {"{sender}": mail.differentEnvelopeSender}) : "",
-										icon: Icons.Warning,
-										helpLink: lang.getInfoLink("mailAuth_link"),
-										buttons: [{text: lang.get("close_alt"), click: () => this._warningDismissed = true}]
-									})
-									: null,
+									helpLink: "mailAuth_link",
+									buttons: [{text: "close_alt", click: () => this._warningDismissed = true}]
+								}) : null,
 							this._renderEventBanner(),
 							this._renderAttachments(),
-							m("hr.hr.mb.mt-s"),
+							// We will only show one banner at a time, with priority given to the mail auth soft fail banner
+							// and fall back to just a separator
```

Comment:
I said I agree but I am strarting to think that they won't find how to show images

Reply:
possible, but I think if you're looking at a message saying the sender couldn't be authenticated then that's not necessarily an issue, idkthough. We can try it to show both